README.md

```diff
@@ -1,2 +1,205 @@
-# Portfolio_setup_and_mobile_first
+# Portfolio setup and mobile first
 Welcome to my portfolio, here I will let you know about my projects.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Kindly remove all comments from your readme file. The comments are just there to guide you on how to fill in the readme file.

Reply:
Hello @Kamba56
All of them? even the comments in every section?

Reply:
> Hello @Kamba56 All of them? even the comments in every section?

Not all the comments please, jist the comments that are there to guide you. Any other comments added by you can renain.